get-started/overview.md

```diff
@@ -69,7 +69,7 @@ business needs dictate, in near real time.
 **Running more workloads on the same hardware**
 
 Docker is lightweight and fast. It provides a viable, cost-effective alternative
-to hypervisor-based virtual machines, so you can use more of your compute
+to hypervisor-based virtual machines, so you can use more of your computer
```

Comment:
Thanks for contributing! 

Looking at this, I think `compute` may have been intentional here ("compute" as in "compute  / computing capacity")

@usha-mandya @dvdksn wdyt? Perhaps there's better wording to prevent confusing?

Reply:
Hey. Thanks for the PR. I agree @thaJeztah! What about something like this?

> It uses fewer resources than hypervisor-based virtual machines, allowing you to dedicate more of your server capacity towards running application code.

Reply:
:-) 
Yes, definitely should be "computing power" or  "capacity". 
Did another commit